src/components/image/test.spec.js

```diff
@@ -22,6 +22,10 @@ describe("Image", () => {
     const tree = renderer.create(<Image src={validFluidSrc} alt="alt ..." />).toJSON()
     expect(tree).toMatchSnapshot()
   })
+  it("will render with a proper fluid image and a default alt attribute", () => {
+    const tree = renderer.create(<Image src={validFluidSrc} />).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
```

Comment:
You're not actually testing the alt attribute here. You're just saying it should match the snapshot. It's not looking at what it sees in the snapshot.

I do think it will be important to look at the rendered output and make sure there is an `alt` attribute in there. But, separately, we should be testing any logic we add. In this case that is the conversion from filename to alt text.